src/modules/graphics/opengl/OpenGL.cpp

```diff
@@ -500,6 +503,19 @@ void OpenGL::initMaxValues()
 		maxShaderStorageBufferBindings = 0;
 	}
 
+	if (GLAD_VERSION_4_3 || GLAD_ES_VERSION_3_1 || GLAD_ARB_compute_shader)
```

Comment:
I think we shouldn't bother with `ARB_compute_shader`, since guaranteeing that every system which supports compute shaders in love also supports GL4.3/ES3.1 makes it more reliable to use, and it won't really leave any hardware behind to do so.